actions/errata.py

```diff
@@ -98,7 +98,12 @@ def old_update(errataidlist, cache_only=None):
 
     return packages.update(packagelist, cache_only)
 
-def update(errataidlist, cache_only=None):
+def update(params, cache_only=None):
+    allow_vendor_change = False
+    if 'allow_vendor_change' in params and params['allow_vendor_change']:
```

Comment:
Old server code has the `params` parameter as `errataidlist` and it is a list.
I think this would crash.

You should first check, if params is a list or a dict.
In case of a list, you have old server code and it is directly the value for `errataidlist`.
Otherwise the code here would work.